src/spyglass/lfp/lfp_merge.py

```diff
@@ -0,0 +1,53 @@
+import datajoint as dj
+from spyglass.lfp.v1 import LFP, ImportedLFP
+
+schema = dj.schema("lfp")
```

Comment:
I believe our schema name was `position_merge`, not sure if this is worth changing since it would create a new table and get rid of existing entries.